---
title: SBOL Visual Specification

# View.
#   1 = List
#   2 = Compact
#   3 = Card
#   4 = Citation
view: 3

# Optional header image (relative to `static/img/` folder).
header:
  caption: ""
  image: ""
---

[Report an Issue with the Current SBOL Visual Specification](https://github.com/SynBioDex/SBOL-visual/issues)

___

[Glyphs]({{< relref "visual-glyphs" >}})

| **3.x.x Release** |  |  |
|---|---|---|
| **3.0.0** <br>The major difference between SBOL Visual 3 and SBOL Visual 2 is that diagrams and glyphs are defined with respect to the SBOL 3 data model rather than the SBOL 2 data model.<br/><br/> A byproduct of this change is that the use of dashed undirected lines for subsystem mappings has been removed. Until a decision is made about how to represent constraints, the specification is mute on both constraints and dashed undirected lines, which means that it is acceptable to use them, if desired, as an annotation indicating identity.<br /><br /> In addition, collection of glyphs has been modified as follows:<br /><ul><li>The deprecated Insulator glyph and “shmoo” Macromolecule alternative glyph have been removed.</li><li>Deprecated BioPAX alternatives to SBO terms for molecular species glyphs have been removed.</li></ul>The changes for this release were approved as SEP V024.</p> |{{% staticref "docs/SBOL-Visual-3.0.pdf" "newtab" %}}Specification{{% /staticref %}} |  |



| **2.x.x Release**  |  |   |
|---|---|---|
| **2.3.0** <br> SBOL Visual 2.3 provides a simple and flexible language for diagrams of genetic systems and design, including both structure (e.g., nucleic acid sequence features) and function (e.g., regulatory interactions and reactions). <br> <br>SBOL Visual 2.3 enhances the prior SBOL Visual 2.2 standard as follows:<br><ul><li>Adds higher-level "interactions with interactions", such as an inducer molecule stimulating a repression interaction</li> <li>Binding with a nucleic acid backbone can be shown by overlapping glyphs, as with other molecular complexes</li> <li>Recommends using polypeptide region glyph to show 2A sequences</li> <li>Adds new "unspecified interaction" glyph for visualizing interactions whose nature is unknown</li> <li>Deprecates "insulator" glyph in favor of new "inert DNA spacer" glyph </li> </ul>|{{% staticref "docs/SBOL-Visual-2.3.pdf" "newtab" %}}Specification{{% /staticref %}}  |   |
| **2.2.0** <br> SBOL Visual 2.2 provides a simple and flexible language for diagrams of genetic systems and design, including both structure (e.g., nucleic acid sequence features) and function (e.g., regulatory interactions and reactions).<br><br> SBOL Visual 2.2 enhances the prior SBOL Visual 2.1 standard as follows: <br> <ul><li> Grounding for molecular species glyphs is changed from BioPAX to SBO (BioPAX terms are retained as deprecated alternatives). </li><li> New glyphs are added for introns and polypeptide regions (e.g., protein domains). </li><li> The macromolecule glyph often confused with yeast is deprecated, and a new glyph for proteins added </li><li> Small polygons are recommended as alternative glyphs for simple chemicals. </li></ul>| {{% staticref "docs/SBOL-Visual-2.2.pdf" "newtab" %}}Specification{{% /staticref %}}  |   |
| **2.1.0** <br> SBOL Visual 2.1 provides a simple and flexible language for diagrams of genetic systems and design, including both structure (e.g., nucleic acid sequence features) and function (e.g., regulatory interactions and reactions). SBOL Visual 2.1 enhances the prior SBOL Visual 2.0 standard with the addition of methods for showing modular structure and mappings between elements of a system, and new glyphs for indicating genomic context (e.g., integration into a plasmid or genome) and for stop codons. Additionally, interactions arrows were modified such that they can split or join, with the glyph at the split or join indicating either superposition or a chemical process.  | {{% staticref "docs/SBOL-Visual-2.1.pdf" "newtab" %}}Specification{{% /staticref %}}  |   |
| **2.0.0** <br> A diagram language and collection of symbols that can be used to graphically depict a genetic design and its functional interactions. SBOL Visual 2.0 builds on the prior SBOL Visual 1.0 standard by expanding diagram syntax to include functional interactions and molecular species, making the relationship between diagrams and the SBOL data model explicit, supporting families of symbol variants, clarifying a number of requirements and best practices, and significantly expanding the collection of diagram glyphs | {{% staticref "docs/SBOLVisual_Specification_2_0_0.pdf" "newtab" %}}Specification{{% /staticref %}}  |  [Cite](https://www.degruyter.com/view/journals/jib/15/1/article-20170074.xml) |

| **1.x.x Release** |  |  |
|---|---|---|
| **1.0.0** <br> A set of symbols that can be used to graphically depict functional information encoded by nucleic acid sequences. Each symbol in the set has three components: a shape, a display name, and a definition. | {{% staticref "docs/BBFRFC93.pdf" "newtab" %}}Specification{{% /staticref %}} | [Cite](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002310) |
