<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.0.0-beta.1 for Hugo"><meta name=author content="Synthetic Biology Open Language"><meta name=description content><link rel=alternate hreflang=en-us href=../publication/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#8cc63f"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin=anonymous><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=../css/wowchemy.2a6125f0583cc2619cb70fca05ed6cff.css><link rel=alternate href=../publication/index.xml type=application/rss+xml title="The Synthetic Biology Open Language"><link rel=manifest href=../index.webmanifest><link rel=icon type=image/png href=../images/icon_hu3e688e08afe16bba9a22fc14ef6ac258_25465_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=../images/icon_hu3e688e08afe16bba9a22fc14ef6ac258_25465_192x192_fill_lanczos_center_2.png><link rel=canonical href=../publication/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="The Synthetic Biology Open Language"><meta property="og:url" content="/publication/"><meta property="og:title" content="Publications | The Synthetic Biology Open Language"><meta property="og:description" content><meta property="og:image" content="/images/logo.svg"><meta property="twitter:image" content="/images/logo.svg"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2021-08-01T00:00:00+00:00"><title>Publications | The Synthetic Biology Open Language</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=../js/wowchemy-init.min.25bb65fc185bceea61395d5b0560127a.js></script><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=../><img src=../images/logo.svg alt="The Synthetic Biology Open Language"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=../><img src=../images/logo.svg alt="The Synthetic Biology Open Language"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=../#about><span>Home</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Data Model</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=../datamodel-about><span>About SBOL</span></a>
<a class=dropdown-item href=../datamodel-specification><span>SBOL Specification</span></a>
<a class=dropdown-item href=../libraries><span>SBOL Libraries</span></a>
<a class=dropdown-item href=../ontology><span>SBOL Ontology</span></a>
<a class=dropdown-item href=https://converter.sbolstandard.org/><span>SBOL Converter</span></a>
<a class=dropdown-item href=https://validator.sbolstandard.org/><span>SBOL Validator</span></a>
<a class=dropdown-item href=../datamodel-examples><span>SBOL Examples</span></a></div></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Visual</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=../visual-about><span>About SBOL Visual</span></a>
<a class=dropdown-item href=../visual-specification><span>SBOL Visual Specification</span></a>
<a class=dropdown-item href=../visual-glyphs><span>SBOL Visual Glyphs</span></a>
<a class=dropdown-item href=../sbol-visual-ontology><span>SBOL Visual Ontology</span></a>
<a class=dropdown-item href=../visual-examples><span>SBOL Visual Examples from papers</span></a>
<a class=dropdown-item href=../visual-spec-examples><span>SBOL Visual Examples from specification</span></a></div></li><li class=nav-item><a class=nav-link href=../applications><span>Applications</span></a></li><li class=nav-item><a class="nav-link active" href=../publication><span>Publications</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Community</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=../community><span>About</span></a>
<a class=dropdown-item href=../community-meetings><span>Meetings</span></a>
<a class=dropdown-item href=../community-outreach><span>Outreach</span></a>
<a class=dropdown-item href=../community-governance><span>Governance</span></a>
<a class=dropdown-item href=../sbol-industrial><span>SBOL Industrial Consortium</span></a></div></li><li class=nav-item><a class=nav-link href=../contact><span>Contact</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Help</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=../faq><span>FAQ</span></a>
<a class=dropdown-item href=https://bioinformatics.stackexchange.com/questions/tagged/sbol><span>StackExchange</span></a></div></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>About</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=https://synbiodex.github.io/SbolStandardWebsite/><span>View Source</span></a>
<a class=dropdown-item href=https://github.com/SynBioDex/Community-Media><span>Community Media</span></a>
<a class=dropdown-item href=https://github.com/SynBioDex/SbolStandardWebsite/issues><span>Report an Issue</span></a></div></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li></ul></div></nav></div><div class=page-body><div class="universal-wrapper pt-3"><h1>Publications</h1></div><div class=universal-wrapper><div class=row><div class=col-lg-12><div class="form-row mb-4"><div class=col-auto><input type=search class=filter-search placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off role=textbox spellcheck=false></div><div class=col-auto><select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group=pubtype>
<option value=*>Type</option>
<option value=.pubtype-0>Uncategorized</option>
<option value=.pubtype-2>Journal article</option>
<option value=.pubtype-6>Book section</option></select></div><div class=col-auto><select class="pub-filters form-control form-control-sm" data-filter-group=year>
<option value=*>Date</option>
<option value=.year-2021>2021</option>
<option value=.year-2020>2020</option>
<option value=.year-2019>2019</option>
<option value=.year-2018>2018</option>
<option value=.year-2017>2017</option>
<option value=.year-2016>2016</option>
<option value=.year-2015>2015</option>
<option value=.year-2014>2014</option>
<option value=.year-2013>2013</option>
<option value=.year-2012>2012</option>
<option value=.year-2011>2011</option>
<option value=.year-2010>2010</option>
<option value=.year-2009>2009</option></select></div></div><div id=container-publications><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class=card-simple><div class=article-metadata><div><span><a href=../author/benjamin-hatch/>Benjamin Hatch</a></span>, <span><a href=../author/linhao-meng/>Linhao Meng</a></span>, <span><a href=../author/jeanet-mante/>Jeanet Mante</a></span>, <span><a href=../author/james-a.-mclaughlin/>James A. McLaughlin</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>August 2021</span>
<span class=middot-divider></span><span class=pub-publication><em>ACS Synthetic Biology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/vis-bol-2-improving-web-based-hatch-2021/>VisBOL2—Improving Web-Based Visualization for Synthetic Biology Designs</a></h3><a href=../publication/vis-bol-2-improving-web-based-hatch-2021/ class=summary-link><div class=article-style><p>VisBOL is a web-based visualization tool used to depict genetic circuit designs. This tool depicts simple DNA circuits adequately, but it has become increasingly outdated as new versions of SBOL Visual were released. This paper introduces VisBOL2, a heavily redesigned version of VisBOL that makes a number of improvements to the original VisBOL, including proper functional interaction rendering, dynamic viewing, a more maintainable code base, and modularity that facilitates compatibility with other software tools. This modularity is demonstrated by incorporating VisBOL2 into a sequence visualization plugin for SynBioHub.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/vis-bol-2-improving-web-based-hatch-2021/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.1c00147 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jeanet-mante/>Jeanet Mante</a></span>, <span><a href=../author/yikai-hao/>Yikai Hao</a></span>, <span><a href=../author/jacob-jett/>Jacob Jett</a></span>, <span><a href=../author/udayan-joshi/>Udayan Joshi</a></span>, <span><a href=../author/kevin-keating/>Kevin Keating</a></span>, <span><a href=../author/xiang-lu/>Xiang Lu</a></span>, <span><a href=../author/gaurav-nakum/>Gaurav Nakum</a></span>, <span><a href=../author/nicholas-e.-rodriguez/>Nicholas E. Rodriguez</a></span>, <span><a href=../author/jiawei-tang/>Jiawei Tang</a></span>, <span><a href=../author/logan-terry/>Logan Terry</a></span>, <span><a href=../author/xuanyu-wu/>Xuanyu Wu</a></span>, <span><a href=../author/eric-yu/>Eric Yu</a></span>, <span><a href=../author/j.-stephen-downie/>J. Stephen Downie</a></span>, <span><a href=../author/bridget-t.-mcinnes/>Bridget T. McInnes</a></span>, <span><a href=../author/mai-h.-nguyen/>Mai H. Nguyen</a></span>, <span><a href=../author/brandon-sepulvado/>Brandon Sepulvado</a></span>, <span><a href=../author/eric-m.-young/>Eric M. Young</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>August 2021</span>
<span class=middot-divider></span><span class=pub-publication><em>ACS Synthetic Biology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/synthetic-biology-knowledge-mante-2021/>Synthetic Biology Knowledge System</a></h3><a href=../publication/synthetic-biology-knowledge-mante-2021/ class=summary-link><div class=article-style><p>The Synthetic Biology Knowledge System (SBKS) is an instance of the SynBioHub repository that includes text and data information that has been mined from papers published in ACS Synthetic Biology. This paper describes the SBKS curation framework that is being developed to construct the knowledge stored in this repository. The text mining pipeline performs automatic annotation of the articles using natural language processing techniques to identify salient content such as key terms, relationships between terms, and main topics. The data mining pipeline performs automatic annotation of the sequences extracted from the supplemental documents with the genetic parts used in them. Together these two pipelines link genetic parts to papers describing the context in which they are used. Ultimately, SBKS will reduce the time necessary for synthetic biologists to find the information necessary to complete their designs.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/synthetic-biology-knowledge-mante-2021/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.1c00188 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class=card-simple><div class=article-metadata><div><span><a href=../author/logan-terry/>Logan Terry</a></span>, <span><a href=../author/jared-earl/>Jared Earl</a></span>, <span><a href=../author/sam-thayer/>Sam Thayer</a></span>, <span><a href=../author/samuel-bridge/>Samuel Bridge</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>June 2021</span>
<span class=middot-divider></span><span class=pub-publication><em>ACS Synthetic Biology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/sbol-canvas-visual-editor-terry-2021/>SBOLCanvas: A Visual Editor for Genetic Designs</a></h3><a href=../publication/sbol-canvas-visual-editor-terry-2021/ class=summary-link><div class=article-style><p>SBOLCanvas is a web-based application that can create and edit genetic constructs using the SBOL data and visual standards. SBOLCanvas allows a user to create a genetic design visually and structurally from start to finish. It also allows users to incorporate existing SBOL data from a SynBioHub repository. By the nature of being a web-based application, SBOLCanvas is readily accessible and easy to use. A live version of the latest release can be found at <a href=https://sbolcanvas.org>https://sbolcanvas.org</a>.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/sbol-canvas-visual-editor-terry-2021/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.1c00096 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/thomas-e.-gorochowski/>Thomas E. Gorochowski</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>April 2020</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/misirli-sbol-2020/>SBOL Visual 2 Ontology</a></h3><a href=../publication/misirli-sbol-2020/ class=summary-link><div class=article-style><p>Standardizing the visual representation of genetic parts and circuits is essential for unambiguously creating and interpreting genetic designs. To this end, an increasing number of tools are adopting well-defined glyphs from the Synthetic Biology Open Language (SBOL) Visual standard to represent various genetic parts and their relationships. However, the implementation and maintenance of the relationships between biological elements or concepts and their associated glyphs has up to now been left up to tool developers. We address this need with the SBOL Visual 2 Ontology, a machine-accessible resource that provides rules for mapping from genetic parts, molecules, and interactions between them, to agreed SBOL Visual glyphs. This resource, together with a web service, can be used as a library to simplify the development of visualization tools, as a stand-alone resource to computationally search for suitable glyphs, and to help facilitate integration with existing biological ontologies and standards in synthetic biology.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.0c00046 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/misirli-sbol-2020/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.0c00046 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/hasan-baig/>Hasan Baig</a></span>, <span><a href=../author/pedro-fontanarrosa/>Pedro Fontanarrosa</a></span>, <span><a href=../author/vishwesh-kulkarni/>Vishwesh Kulkarni</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/prashant-vaidyanathan/>Prashant Vaidyanathan</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/shyam-bhakta/>Shyam Bhakta</a></span>, <span><a href=../author/michael-bissell/>Michael Bissell</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/robert-sidney-cox/>Robert Sidney Cox</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/thomas-gorochowski/>Thomas Gorochowski</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/augustin-luna/>Augustin Luna</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/nicolas-le-novere/>Nicolas Le Novère</a></span>, <span><a href=../author/zachary-palchick/>Zachary Palchick</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/john-t-sexton/>John T Sexton</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/jeffrey-j-tabor/>Jeffrey J Tabor</a></span>, <span><a href=../author/marta-vazquez-vilar/>Marta Vazquez Vilar</a></span>, <span><a href=../author/christopher-a-voigt/>Christopher A Voigt</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/david-zong/>David Zong</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span></div><span class=article-date>April 2020</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/visualsbol221/>Synthetic biology open language visual (SBOL visual) version 2.2</a></h3><a href=../publication/visualsbol221/ class=summary-link><div class=article-style><p>People who are engineering biological organisms often find it useful to communicate in diagrams, both about the structure of the nucleic acid sequences that they are engineering and about the functional relationships between sequence features and other molecular species. Some typical practices and conventions have begun to emerge for such diagrams. The Synthetic Biology Open Language Visual (SBOL Visual) has been developed as a standard for organizing and systematizing such conventions in order to produce a coherent language for expressing the structure and function of genetic designs. This document details version 2.2 of SBOL Visual, which builds on the prior SBOL Visual 2.1 in several ways. First, the grounding of molecular species glyphs is changed from BioPAX to SBO, aligning with the use of SBO terms for interaction glyphs. Second, new glyphs are added for proteins, introns, and polypeptide regions (e. g., protein domains), the prior recommended macromolecule glyph is deprecated in favor of its alternative, and small polygons are introduced as alternative glyphs for simple chemicals.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/visualsbol221/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/hasan-baig/>Hasan Baig</a></span>, <span><a href=../author/pedro-fontanarrosa/>Pedro Fontanarrosa</a></span>, <span><a href=../author/vishwesh-kulkarni/>Vishwesh Kulkarni</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/prashant-vaidyanathan/>Prashant Vaidyanathan</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/matthew-crowther/>Matthew Crowther</a></span>, <span><a href=../author/thomas-gorochowski/>Thomas Gorochowski</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>April 2020</span>
<span class=middot-divider></span><span class=pub-publication></span></div><a href=../publication/synthetic-biology-open-baig-2020/><img src=../publication/synthetic-biology-open-baig-2020/featured_hu2c94b37cff1b36e13c29b86e50d08051_143131_600x400_fit_lanczos_2.png class=article-banner alt="Synthetic biology open language (SBOL) version 3.0.0"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/synthetic-biology-open-baig-2020/>Synthetic biology open language (SBOL) version 3.0.0</a></h3><a href=../publication/synthetic-biology-open-baig-2020/ class=summary-link><div class=article-style><p>Synthetic biology builds upon genetics, molecular biology, and metabolic engineering by applying engineering principles to the design of biological systems. When designing a synthetic system, synthetic biologists need to exchange information about multiple types of molecules, the intended behavior of the system, and actual experimental measurements. The Synthetic Biology Open Language (SBOL) has been developed as a standard to support the specification and exchange of biological design information in synthetic biology, following an open community process involving both wet bench scientists and dry scientific modelers and software developers, across academia, industry, and other institutions. This document describes SBOL 3.0.0, which condenses and simplifies previous versions of SBOL based on experiences in deployment across a variety of scientific and industrial settings. In particular, SBOL 3.0.0, (1) separates sequence features from part/sub-part relationships, (2) renames Component Definition/Component to Component/Sub-Component, (3) merges Component and Module classes, (4) ensures consistency between data model and ontology terms, (5) extends the means to define and reference Sub-Components, (6) refines requirements on object URIs, (7) enables graph-based serialization, (8) moves Systems Biology Ontology (SBO) for Component types, (9) makes all sequence associations explicit, (10) makes interfaces explicit, (11) generalizes Sequence Constraints into a general structural Constraint class, and (12) expands the set of allowed constraints.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/synthetic-biology-open-baig-2020/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/thomas-e-gorochowski/>Thomas E Gorochowski</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/chris-j-myers/>Chris J Myers</a></span></div><span class=article-date>March 2020</span>
<span class=middot-divider></span><span class=pub-publication></span></div><a href=../publication/visual2ontology/><img src=../publication/visual2ontology/featured_hu9efc1259ea444fcb9266c952d06f7b2c_74272_600x400_fit_q90_lanczos.jpeg class=article-banner alt="SBOL Visual 2 Ontology"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/visual2ontology/>SBOL Visual 2 Ontology</a></h3><a href=../publication/visual2ontology/ class=summary-link><div class=article-style><p>Standardizing the visual representation of genetic parts and circuits is essential for unambiguously creating and interpreting genetic designs. To this end, an increasing number of tools are adopting well-defined glyphs from the Synthetic Biology Open Language (SBOL) Visual standard to represent various genetic parts and their relationships. However, the implementation and maintenance of the relationships between biological elements or concepts and their associated glyphs has up to now been left up to tool developers. We address this need with the SBOL Visual 2 Ontology, a machine-accessible resource that provides rules for mapping from genetic parts, molecules, and interactions between them, to agreed SBOL Visual glyphs. This resource, together with a web service, can be used as a library to simplify the development of visualization tools, as a stand-alone resource to computationally search for suitable glyphs, and to help facilitate integration with existing biological ontologies and standards in synthetic biology.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/visual2ontology/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/matthew-crowther/>Matthew Crowther</a></span>, <span><a href=../author/lewis-grozinger/>Lewis Grozinger</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/christopher-p.-d.-taylor/>Christopher P. D. Taylor</a></span>, <span><a href=../author/james-a.-mclaughlin/>James A. McLaughlin</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/bryan-a.-bartley/>Bryan A. Bartley</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>March 2020</span>
<span class=middot-divider></span><span class=pub-publication></span></div><a href=../publication/shortbol/><img src=../publication/shortbol/featured_huf8ba585f3f0df5edfa9fad214d548dff_51938_600x400_fit_q90_lanczos.jpg class=article-banner alt="ShortBOL: A Language for Scripting Designs for Engineered Biological Systems Using Synthetic Biology Open Language (SBOL)"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/shortbol/>ShortBOL: A Language for Scripting Designs for Engineered Biological Systems Using Synthetic Biology Open Language (SBOL)</a></h3><a href=../publication/shortbol/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is an emerging synthetic biology data exchange standard, designed primarily for unambiguous and efficient machine-to-machine communication. However, manual editing of SBOL is generally difficult for nontrivial designs. Here, we describe ShortBOL, a lightweight SBOL scripting language that bridges the gap between manual editing, visual design tools, and direct programming. ShortBOL is a shorthand textual language developed to enable users to create SBOL designs quickly and easily, without requiring strong programming skills or visual design tools.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/shortbol/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class=card-simple><div class=article-metadata><div><span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/bryan-a.-bartley/>Bryan A. Bartley</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/prashant-vaidyanathan/>Prashant Vaidyanathan</a></span>, <span><a href=../author/pedro-fontanarrosa/>Pedro Fontanarrosa</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/thomas-e.-gorochowski/>Thomas E. Gorochowski</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>January 2020</span>
<span class=middot-divider></span><span class=pub-publication><em>Frontiers in Bioengineering and Biotechnology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/synthetic-biology-open-mclaughlin-2020-a/>The Synthetic Biology Open Language (SBOL) Version 3: Simplified Data Exchange for Bioengineering</a></h3><a href=../publication/synthetic-biology-open-mclaughlin-2020-a/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is a community-developed data standard that allows knowledge about biological designs to be captured using a machine-tractable, ontology-backed representation that is built using Semantic Web technologies. While early versions of SBOL focused only on the description of DNA-based components and their sub-components, SBOL can now be used to represent knowledge across multiple scales and throughout the entire synthetic biology workflow, from the specification of a single molecule or DNA fragment through to multicellular systems containing multiple interacting genetic circuits. The third major iteration of the SBOL standard, SBOL3, is an effort to streamline and simplify the underlying data model with a focus on real-world applications, based on experience from the deployment of SBOL in a variety of scientific and industrial settings. Here, we introduce the SBOL3 specification both in comparison to previous versions of SBOL and through practical examples of its use.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/synthetic-biology-open-mclaughlin-2020-a/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.3389/fbioe.2020.01009 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/thomas-e.-gorochowski/>Thomas E. Gorochowski</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/benjamin-aleritsch/>Benjamin Aleritsch</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/shyam-bhakta/>Shyam Bhakta</a></span>, <span><a href=../author/mike-bissell/>Mike Bissell</a></span>, <span><a href=../author/sebastian-castillo-hair/>Sebastian Castillo Hair</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/augustin-luna/>Augustin Luna</a></span>, <span><a href=../author/nicolas-le-novere/>Nicolas Le Novère</a></span>, <span><a href=../author/zach-palchick/>Zach Palchick</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/john-t.-sexton/>John T. Sexton</a></span>, <span><a href=../author/jeffrey-j.-tabor/>Jeffrey J. Tabor</a></span>, <span><a href=../author/christopher-a.-voigt/>Christopher A. Voigt</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>August 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/beal-communicating-2019/>Communicating Structure and Function in Synthetic Biology Diagrams</a></h3><a href=../publication/beal-communicating-2019/ class=summary-link><div class=article-style><p>Biological engineers often find it useful to communicate using diagrams. These diagrams can include information both about the structure of the nucleic acid sequences they are engineering and about the functional relationships between features of these sequences and/or other molecular species. A number of conventions and practices have begun to emerge within synthetic biology for creating such diagrams, and the Synthetic Biology Open Language Visual (SBOL Visual) has been developed as a standard to organize, systematize, and extend such conventions in order to produce a coherent visual language. Here, we describe SBOL Visual version 2, which expands previous diagram standards to include new functional interactions, categories of molecular species, support for families of glyph variants, and the ability to indicate modular structure and mappings between elements of a system. SBOL Visual 2 also clarifies a number of requirements and best practices, significantly expands the collection of glyphs available to describe genetic features, and can be readily applied using a wide variety of software tools, both general and bespoke.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.9b00139 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/beal-communicating-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.9b00139 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>July 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><a href=../publication/roehner-specifying-2019/><img src=../publication/roehner-specifying-2019/featured_hu9fbb00f34284a9fb6de662bba6bf5f6a_343726_600x400_fit_lanczos_2.png class=article-banner alt="Specifying Combinatorial Designs with the Synthetic Biology Open Language (SBOL)"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/roehner-specifying-2019/>Specifying Combinatorial Designs with the Synthetic Biology Open Language (SBOL)</a></h3><a href=../publication/roehner-specifying-2019/ class=summary-link><div class=article-style><p>As improvements in DNA synthesis technology and assembly methods make combinatorial assembly of genetic constructs increasingly accessible, methods for representing genetic constructs likewise need to improve to handle the exponential growth of combinatorial design space. To this end, we present a community accepted extension of the SBOL data standard that allows for the efficient and flexible encoding of combinatorial designs. This extension includes data structures for representing genetic designs with “variable” components that can be implemented by choosing one of many linked designs for existing genetic parts or constructs. We demonstrate the representational power of the SBOL combinatorial design extension through case studies on metabolic pathway design and genetic circuit design, and we report the expansion of the SBOLDesigner software tool to support users in creating and modifying combinatorial designs in SBOL.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.9b00092 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/roehner-specifying-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.9b00092 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/renee-taylor/>Renee Taylor</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/phillip-lord/>Phillip Lord</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>July 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/misirli-sbol-owl-2019/>SBOL-OWL: An Ontological Approach for Formal and Semantic Representation of Synthetic Biology Information</a></h3><a href=../publication/misirli-sbol-owl-2019/ class=summary-link><div class=article-style><p>Standard representation of data is key for the reproducibility of designs in synthetic biology. The Synthetic Biology Open Language (SBOL) has already emerged as a data standard to represent information about genetic circuits, and it is based on capturing data using graphs. The language provides the syntax using a free text document that is accessible to humans only. This paper describes SBOL-OWL, an ontology for a machine understandable definition of SBOL. This ontology acts as a semantic layer for genetic circuit designs. As a result, computational tools can understand the meaning of design entities in addition to parsing structured SBOL data. SBOL-OWL not only describes how genetic circuits can be constructed computationally, it also facilitates the use of several existing Semantic Web tools for synthetic biology. This paper demonstrates some of these features, for example, to validate designs and check for inconsistencies. Through the use of SBOL-OWL, queries can be simplified and become more intuitive. Moreover, existing reasoners can be used to infer information about genetic circuit designs that cannot be directly retrieved using existing querying mechanisms. This ontological representation of the SBOL standard provides a new perspective to the verification, representation, and querying of information about genetic circuits and is important to incorporate complex design information via the integration of biological ontologies.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00532 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/misirli-sbol-owl-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00532 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/bryan-a.-bartley/>Bryan A. Bartley</a></span>, <span><a href=../author/kiri-choi/>Kiri Choi</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>July 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/bartley-pysbol-2019/>pySBOL: A Python Package for Genetic Design Automation and Standardization</a></h3><a href=../publication/bartley-pysbol-2019/ class=summary-link><div class=article-style><p>This paper presents pySBOL, a software library for computer-aided design of synthetic biological systems in the Python scripting language. This library provides an easy-to-use, object-oriented, application programming interface (API) with low barrier of entry for synthetic biology application developers. The pySBOL library enables reuse of genetic parts and designs through standardized data exchange with biological parts repositories and software tools that communicate using the Synthetic Biology Open Language (SBOL). In addition, pySBOL supports data management of design-build-test-learn workflows for individual laboratories as well as large, distributed teams of synthetic biologists. PySBOL also lets users add custom data to SBOL files to support the specific data requirements of their research. This extensibility helps users integrate software tool chains and develop workflows for new applications. These features and others make the pySBOL library a valuable tool for supporting engineering practices in synthetic biology. Documentation and installation instructions can be found at pysbol2.readthedocs.io.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00336 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/bartley-pysbol-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00336 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/prashant-vaidyanathan/>Prashant Vaidyanathan</a></span>, <span><a href=../author/timothy-s.-jones/>Timothy S. Jones</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>July 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/misirli-computational-2019/>A Computational Workflow for the Automated Generation of Models of Genetic Designs</a></h3><a href=../publication/misirli-computational-2019/ class=summary-link><div class=article-style><p>Computational models are essential to engineer predictable biological systems and to scale up this process for complex systems. Computational modeling often requires expert knowledge and data to build models. Clearly, manual creation of models is not scalable for large designs. Despite several automated model construction approaches, computational methodologies to bridge knowledge in design repositories and the process of creating computational models have still not been established. This paper describes a workflow for automatic generation of computational models of genetic circuits from data stored in design repositories using existing standards. This workflow leverages the software tool SBOLDesigner to build structural models that are then enriched by the Virtual Parts Repository API using Systems Biology Open Language (SBOL) data fetched from the SynBioHub design repository. The iBioSim software tool is then utilized to convert this SBOL description into a computational model encoded using the Systems Biology Markup Language (SBML). Finally, this SBML model can be simulated using a variety of methods. This workflow provides synthetic biologists with easy to use tools to create predictable biological systems, hiding away the complexity of building computational models. This approach can further be incorporated into other computational workflows for design automation.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.7b00459 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/misirli-computational-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.7b00459 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/timothy-s.-jones/>Timothy S. Jones</a></span>, <span><a href=../author/pedro-fontanarrosa/>Pedro Fontanarrosa</a></span>, <span><a href=../author/jeanet-v.-mante/>Jeanet V. Mante</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>July 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/nguyen-design-2019/>Design of Asynchronous Genetic Circuits</a></h3><a href=../publication/nguyen-design-2019/ class=summary-link><div class=article-style><p>Most digital electronic circuits utilize a timing reference to synchronize the progression of signals and enable sequential memory elements. These designs may not be realizable in biological substrates due to the lack of a reliable high-frequency clock signal. Asynchronous designs eliminate the need for a clock with data encodings and request/acknowledge handshake protocols. This paper proposes a workflow to automate the design of asynchronous genetic circuits. This workflow extends genetic design tools by leveraging asynchronous logic design methods customized for this technology. This workflow is demonstrated on a genetic sensor that uses filtering and cellular communication to improve its reliability.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/nguyen-design-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1109/JPROC.2019.2916057 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/zachary-palchick/>Zachary Palchick</a></span>, <span><a href=../author/umesh-p/>Umesh P</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/shyam-bhakta/>Shyam Bhakta</a></span>, <span><a href=../author/mike-bissell/>Mike Bissell</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/robert-sidney-cox/>Robert Sidney Cox</a></span>, <span><a href=../author/thomas-gorochowski/>Thomas Gorochowski</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/augustin-luna/>Augustin Luna</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/nicolas-le-novere/>Nicolas Le Novère</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/john-t.-sexton/>John T. Sexton</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/jeffrey-j.-tabor/>Jeffrey J. Tabor</a></span>, <span><a href=../author/christopher-a.-voigt/>Christopher A. Voigt</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>June 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/madsen-synthetic-2019/>Synthetic Biology Open Language Visual (SBOL Visual) Version 2.1</a></h3><a href=../publication/madsen-synthetic-2019/ class=summary-link><div class=article-style><p>Abstract People who are engineering biological organisms often find it useful to communicate in diagrams, both about the structure of the nucleic acid sequences that they are engineering and about the functional relationships between sequence features and other molecular species . Some typical practices and conventions have begun to emerge for such diagrams. The Synthetic Biology Open Language Visual (SBOL Visual) has been developed as a standard for organizing and systematizing such conventions in order to produce a coherent language for expressing the structure and function of genetic designs. This document details version 2.1 of SBOL Visual, which builds on the prior SBOL Visual 2.0 standard by expanding diagram syntax to include methods for showing modular structure and mappings between elements of a system, interactions arrows that can split or join (with the glyph at the split or join indicating either superposition or a chemical process), and adding new glyphs for indicating genomic context (e.g., integration into a plasmid or genome) and for stop codons.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.degruyter.com/view/journals/jib/16/2/article-20180101.xml target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/madsen-synthetic-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1515/jib-2018-0101 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/umesh-p/>Umesh P</a></span>, <span><a href=../author/zachary-palchick/>Zachary Palchick</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/christian-atallah/>Christian Atallah</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/kiri-choi/>Kiri Choi</a></span>, <span><a href=../author/robert-sidney-cox/>Robert Sidney Cox</a></span>, <span><a href=../author/thomas-gorochowski/>Thomas Gorochowski</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/chris-macklin/>Chris Macklin</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/xianwei-meng/>Xianwei Meng</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/ysis-tarter/>Ysis Tarter</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/michael-bissell/>Michael Bissell</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>June 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/madsen-synthetic-2019-1/>Synthetic Biology Open Language (SBOL) Version 2.3</a></h3><a href=../publication/madsen-synthetic-2019-1/ class=summary-link><div class=article-style><p>textlesssection class="abstract"textgreatertextlessh2 class="abstractTitle text-title my-1" id="d230e2"textgreaterAbstracttextless/h2textgreatertextlessptextgreaterSynthetic biology builds upon the techniques and successes of genetics, molecular biology, and metabolic engineering by applying engineering principles to the design of biological systems. The field still faces substantial challenges, including long development times, high rates of failure, and poor reproducibility. One method to ameliorate these problems is to improve the exchange of information about designed systems between laboratories. The synthetic biology open language (SBOL) has been developed as a standard to support the specification and exchange of biological design information in synthetic biology, filling a need not satisfied by other pre-existing standards. This document details version 2.3.0 of SBOL, which builds upon version 2.2.0 published in last year’s JIB Standards in Systems Biology special issue. In particular, SBOL 2.3.0 includes means of succinctly representing sequence modifications, such as insertion, deletion, and replacement, an extension to support organization and attachment of experimental data derived from designs, and an extension for describing numerical parameters of design elements. The new version also includes specifying types of synthetic biology activities, unambiguous locations for sequences with multiple encodings, refinement of a number of validation rules, improved figures and examples, and clarification on a number of issues related to the use of external ontology terms.textless/ptextgreatertextless/sectiontextgreater</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.degruyter.com/view/journals/jib/16/2/article-20190025.xml target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/madsen-synthetic-2019-1/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1515/jib-2019-0025 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/nathan-wilkinson/>Nathan Wilkinson</a></span>, <span><a href=../author/christian-atallah/>Christian Atallah</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>January 2019</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/mclaughlin-sboljs-2019/>sboljs: Bringing the Synthetic Biology Open Language to the Web Browser</a></h3><a href=../publication/mclaughlin-sboljs-2019/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is a data standard for the representation of engineered biological systems. SBOL is implemented in the form of software libraries which can be used to add SBOL support to both new and existing software tools. While existing libraries allow for software to be developed that runs on a server or is installed locally, they lack the capability to create SBOL software that runs directly in a Web browser. Here, we address this issue by presenting sboljs, a JavaScript software library for SBOL that is capable of being used both on the server and in the Web browser.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00338 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/mclaughlin-sboljs-2019/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.8b00338 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/timothy-s-jones/>Timothy S Jones</a></span>, <span><a href=../author/pedro-fontanarrosa/>Pedro Fontanarrosa</a></span>, <span><a href=../author/jeanet-v-mante/>Jeanet V Mante</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/chris-j-myers/>Chris J Myers</a></span></div><span class=article-date>January 2019</span>
<span class=middot-divider></span><span class=pub-publication><em>Proceedings of the IEEE</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/nguyen-2019-design/>Design of asynchronous genetic circuits</a></h3><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/nguyen-2019-design/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/thomas-e-gorochowski/>Thomas E Gorochowski</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/james-scott-brown/>James Scott-Brown</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/benjamin-aleritsch/>Benjamin Aleritsch</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/shyam-bhakta/>Shyam Bhakta</a></span>, <span><a href=../author/others/>others</a></span></div><span class=article-date>January 2019</span>
<span class=middot-divider></span><span class=pub-publication><em>ACS synthetic biology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/beal-2019-communicating/>Communicating structure and function in synthetic biology diagrams</a></h3><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/beal-2019-communicating/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018"><div class=card-simple><div class=article-metadata><div><span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/irina-dana-ofiteru/>Irina Dana Ofiteru</a></span>, <span><a href=../author/angel-goni-moreno/>Angel Goñi-Moreno</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>February 2018</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/mclaughlin-synbiohub-2018/>SynBioHub: A Standards-Enabled Design Repository for Synthetic Biology</a></h3><a href=../publication/mclaughlin-synbiohub-2018/ class=summary-link><div class=article-style><p>The SynBioHub repository (<a href=https://synbiohub.org>https://synbiohub.org</a>) is an open-source software project that facilitates the sharing of information about engineered biological systems. SynBioHub provides computational access for software and data integration, and a graphical user interface that enables users to search for and share designs in a Web browser. By connecting to relevant repositories (e.g., the iGEM repository, JBEI ICE, and other instances of SynBioHub), the software allows users to browse, upload, and download data in various standard formats, regardless of their location or representation. SynBioHub also provides a central reference point for other resources to link to, delivering design information in a standardized format using the Synthetic Biology Open Language (SBOL). The adoption and use of SynBioHub, a community-driven effort, has the potential to overcome the reproducibility challenge across laboratories by helping to address the current lack of information about published designs.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.7b00403 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/mclaughlin-synbiohub-2018/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.7b00403 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>July 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/zhang-sboldesigner-2017/>SBOLDesigner 2: An Intuitive Tool for Structural Genetic Design</a></h3><a href=../publication/zhang-sboldesigner-2017/ class=summary-link><div class=article-style><p>As the Synthetic Biology Open Language (SBOL) data and visual standards gain acceptance for describing genetic designs in a detailed and reproducible way, there is an increasing need for an intuitive sequence editor tool that biologists can use that supports these standards. This paper describes SBOLDesigner 2, a genetic design automation (GDA) tool that natively supports both the SBOL data model (Version 2) and SBOL Visual (Version 1). This software is enabled to fetch and store parts and designs from SBOL repositories, such as SynBioHub. It can also import and export data about parts and designs in FASTA, GenBank, and SBOL 1 data format. Finally, it possesses a simple and intuitive user interface. This paper describes the design process using SBOLDesigner 2, highlighting new features over the earlier prototype versions. SBOLDesigner 2 is released freely and open source under the Apache 2.0 license.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00275 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/zhang-sboldesigner-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00275 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/robert-sidney-cox/>Robert Sidney Cox</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>July 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/cox-visual-2017/>A Visual Language for Protein Design</a></h3><a href=../publication/cox-visual-2017/ class=summary-link><div class=article-style><p>As protein engineering becomes more sophisticated, practitioners increasingly need to share diagrams for communicating protein designs. To this end, we present a draft visual language, Protein Language, that describes the high-level architecture of an engineered protein with easy-to-draw glyphs, intended to be compatible with other biological diagram languages such as SBOL Visual and SBGN. Protein Language consists of glyphs for representing important features (e.g., globular domains, recognition and localization sequences, sites of covalent modification, cleavage and catalysis), rules for composing these glyphs to represent complex architectures, and rules constraining the scaling and styling of diagrams. To support Protein Language we have implemented an extensible web-based software diagram tool, Protein Designer, that uses Protein Language in a “drag and drop” interface for visualization and computer-aided-design of engineered proteins, as well as conversion of annotated protein sequences to Protein Language diagrams and figure export. Protein Designer can be accessed at <a href=http://protlang.biocad.io/>http://protlang.biocad.io/</a>.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00286 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/cox-visual-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00286 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>July 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/zundel-validator-2017/>A Validator and Converter for the Synthetic Biology Open Language</a></h3><a href=../publication/zundel-validator-2017/ class=summary-link><div class=article-style><p>This paper presents a new validation and conversion utility for the Synthetic Biology Open Language (SBOL). This utility can be accessed directly in software using the libSBOLj library, through a web interface, or using a web service via RESTful API calls. The validator checks all required and best practice rules set forth in the SBOL specification document, and it reports back to the user the location within the document of any errors found. The converter is capable of translating from/to SBOL 1, GenBank, and FASTA formats to/from SBOL 2. The SBOL Validator/Converter utility is released freely and open source under the Apache 2.0 license. The online version of the validator/converter utility can be found here: <a href=http://www.async.ece.utah.edu/sbol-validator/>http://www.async.ece.utah.edu/sbol-validator/</a>. The source code for the validator/converter can be found here: <a href=http://github.com/SynBioDex/SBOL-Validator/>http://github.com/SynBioDex/SBOL-Validator/</a>.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00277 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/zundel-validator-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00277 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/thomas-e.-gorochowski/>Thomas E. Gorochowski</a></span>, <span><a href=../author/hiroyuki-kuwahara/>Hiroyuki Kuwahara</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span></div><span class=article-date>June 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/myers-standard-enabled-2017/>A standard-enabled workflow for synthetic biology</a></h3><a href=../publication/myers-standard-enabled-2017/ class=summary-link><div class=article-style><p>A synthetic biology workflow is composed of data repositories that provide information about genetic parts, sequence-level design tools to compose these parts into circuits, visualization tools to depict these designs, genetic design tools to select parts to create systems, and modeling and simulation tools to evaluate alternative design choices. Data standards enable the ready exchange of information within such a workflow, allowing repositories and tools to be connected from a diversity of sources. The present paper describes one such workflow that utilizes, among others, the Synthetic Biology Open Language (SBOL) to describe genetic designs, the Systems Biology Markup Language to model these designs, and SBOL Visual to visualize these designs. We describe how a standard-enabled workflow can be used to produce types of design information, including multiple repositories and software tools exchanging information using a variety of data standards. Recently, the ACS Synthetic Biology journal has recommended the use of SBOL in their publications.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://portlandpress.com/biochemsoctrans/article-abstract/45/3/793/66896/A-standard-enabled-workflow-for-synthetic-biology?redirectedFrom=fulltext" target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/myers-standard-enabled-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1042/BST20160347 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/hiroyuki-kuwahara/>Hiroyuki Kuwahara</a></span>, <span><a href=../author/xuefeng-cui/>Xuefeng Cui</a></span>, <span><a href=../author/ramzan-umarov/>Ramzan Umarov</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/xin-gao/>Xin Gao</a></span></div><span class=article-date>April 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/kuwahara-sbolme-2017/>SBOLme: a Repository of SBOL Parts for Metabolic Engineering</a></h3><a href=../publication/kuwahara-sbolme-2017/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is a community-driven open language to promote standardization in synthetic biology. To support the use of SBOL in metabolic engineering, we developed SBOLme, the first open-access repository of SBOL 2-compliant biochemical parts for a wide range of metabolic engineering applications. The URL of our repository is <a href=http://www.cbrc.kaust.edu.sa/sbolme>http://www.cbrc.kaust.edu.sa/sbolme</a>.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00278 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/kuwahara-sbolme-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00278 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017"><div class=card-simple><div class=article-metadata><div><span><a href=../author/herbert-sauro/>Herbert Sauro</a></span></div><span class=article-date>March 2017</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/sauro-synthetic-2017/>Synthetic biology: Enter the living machine</a></h3><a href=../publication/sauro-synthetic-2017/ class=summary-link><div class=article-style><p>Herbert Sauro delves into a history of attempts to engineer life.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/543178a target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/sauro-synthetic-2017/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/543178a target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/jennifer-hallinan/>Jennifer Hallinan</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/phillip-lord/>Phillip Lord</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>October 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/misirli-data-2016/>Data Integration and Mining for Synthetic Biology Design</a></h3><a href=../publication/misirli-data-2016/ class=summary-link><div class=article-style><p>One aim of synthetic biologists is to create novel and predictable biological systems from simpler modular parts. This approach is currently hampered by a lack of well-defined and characterized parts and devices. However, there is a wealth of existing biological information, which can be used to identify and characterize biological parts, and their design constraints in the literature and numerous biological databases. However, this information is spread among these databases in many different formats. New computational approaches are required to make this information available in an integrated format that is more amenable to data mining. A tried and tested approach to this problem is to map disparate data sources into a single data set, with common syntax and semantics, to produce a data warehouse or knowledge base. Ontologies have been used extensively in the life sciences, providing this common syntax and semantics as a model for a given biological domain, in a fashion that is amenable to computational analysis and reasoning. Here, we present an ontology for applications in synthetic biology design, SyBiOnt, which facilitates the modeling of information about biological parts and their relationships. SyBiOnt was used to create the SyBiOntKB knowledge base, incorporating and building upon existing life sciences ontologies and standards. The reasoning capabilities of ontologies were then applied to automate the mining of biological parts from this knowledge base. We propose that this approach will be useful to speed up synthetic biology design and ultimately help facilitate the automation of the biological engineering life cycle.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00295 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/misirli-data-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00295 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/robert-sidney-cox/>Robert Sidney Cox</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/james-mclaughlin/>James McLaughlin</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/michael-bissell/>Michael Bissell</a></span>, <span><a href=../author/kiri-choi/>Kiri Choi</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/chris-macklin/>Chris Macklin</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>September 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/beal-synthetic-2016/>Synthetic Biology Open Language (SBOL) Version 2.1.0</a></h3><a href=../publication/beal-synthetic-2016/ class=summary-link><div class=article-style><p>textlesssection class="abstract"textgreatertextlessh2 class="abstractTitle text-title my-1" id="d807e2"textgreaterSummarytextless/h2textgreatertextlessptextgreaterSynthetic biology builds upon the techniques and successes of genetics, molecular biology, and metabolic engineering by applying engineering principles to the design of biological systems. The field still faces substantial challenges, including long development times, high rates of failure, and poor reproducibility. One method to ameliorate these problems would be to improve the exchange of information about designed systems between laboratories. The Synthetic Biology Open Language (SBOL) has been developed as a standard to support the specification and exchange of biological design information in synthetic biology, filling a need not satisfied by other pre-existing standards. This document details version 2.1 of SBOL that builds upon version 2.0 published in last year’s JIB special issue. In particular, SBOL 2.1 includes improved rules for what constitutes a valid SBOL document, new role fields to simplify the expression of sequence features and how components are used in context, and new best practices descriptions to improve the exchange of basic sequence topology information and the description of genetic design provenance, as well as miscellaneous other minor improvements.textless/ptextgreatertextless/sectiontextgreater</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.degruyter.com/view/journals/jib/13/3/article-p30.xml target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/beal-synthetic-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1515/jib-2016-291 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>August 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/mclaughlin-visbol-2016/>VisBOL: Web-Based Tools for Synthetic Biology Design Visualization</a></h3><a href=../publication/mclaughlin-visbol-2016/ class=summary-link><div class=article-style><p>VisBOL is a Web-based application that allows the rendering of genetic circuit designs, enabling synthetic biologists to visually convey designs in SBOL visual format. VisBOL designs can be exported to formats including PNG and SVG images to be embedded in Web pages, presentations and publications. The VisBOL tool enables the automated generation of visualizations from designs specified using the Synthetic Biology Open Language (SBOL) version 2.0, as well as a range of well-known bioinformatics formats including GenBank and Pigeoncad notation. VisBOL is provided both as a user accessible Web site and as an open-source (BSD) JavaScript library that can be used to embed diagrams within other content and software.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00244 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/mclaughlin-visbol-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00244 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/goksel-m%C4%B1s%C4%B1rl%C4%B1/>Göksel Mısırlı</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/keith-flanagan/>Keith Flanagan</a></span>, <span><a href=../author/jennifer-hallinan/>Jennifer Hallinan</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>June 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/madsen-sbol-2016/>The SBOL Stack: A Platform for Storing, Publishing, and Sharing Synthetic Biology Designs</a></h3><a href=../publication/madsen-sbol-2016/ class=summary-link><div class=article-style><p>Recently, synthetic biologists have developed the Synthetic Biology Open Language (SBOL), a data exchange standard for descriptions of genetic parts, devices, modules, and systems. The goals of this standard are to allow scientists to exchange designs of biological parts and systems, to facilitate the storage of genetic designs in repositories, and to facilitate the description of genetic designs in publications. In order to achieve these goals, the development of an infrastructure to store, retrieve, and exchange SBOL data is necessary. To address this problem, we have developed the SBOL Stack, a Resource Description Framework (RDF) database specifically designed for the storage, integration, and publication of SBOL data. This database allows users to define a library of synthetic parts and designs as a service, to share SBOL data with collaborators, and to store designs of biological systems locally. The database also allows external data sources to be integrated by mapping them to the SBOL data model. The SBOL Stack includes two Web interfaces: the SBOL Stack API and SynBioHub. While the former is designed for developers, the latter allows users to upload new SBOL biological designs, download SBOL documents, search by keyword, and visualize SBOL data. Since the SBOL Stack is based on semantic Web technology, the inherent distributed querying functionality of RDF databases can be used to allow different SBOL stack databases to be queried simultaneously, and therefore, data can be shared between different institutes, centers, or other users.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00210 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/madsen-sbol-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00210 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/michael-bissell/>Michael Bissell</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/michael-zhang/>Michael Zhang</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>June 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/roehner-sharing-2016/>Sharing Structure and Function in Biological Design with SBOL 2.0</a></h3><a href=../publication/roehner-sharing-2016/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is a standard that enables collaborative engineering of biological systems across different institutions and tools. SBOL is developed through careful consideration of recent synthetic biology trends, real use cases, and consensus among leading researchers in the field and members of commercial biotechnology enterprises. We demonstrate and discuss how a set of SBOL-enabled software tools can form an integrated, cross-organizational workflow to recapitulate the design of one of the largest published genetic circuits to date, a 4-input AND sensor. This design encompasses the structural components of the system, such as its DNA, RNA, small molecules, and proteins, as well as the interactions between these components that determine the system’s behavior/function. The demonstrated workflow and resulting circuit design illustrate the utility of SBOL 2.0 in automating the exchange of structural and functional specifications for genetic parts, devices, and the biological systems in which they operate.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00215 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/roehner-sharing-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00215 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nathan-j.-hillson/>Nathan J. Hillson</a></span>, <span><a href=../author/hector-a.-plahar/>Hector A. Plahar</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/ranjini-prithviraj/>Ranjini Prithviraj</a></span></div><span class=article-date>June 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/hillson-improving-2016/>Improving Synthetic Biology Communication: Recommended Practices for Visual Depiction and Digital Submission of Genetic Designs</a></h3><a href=../publication/hillson-improving-2016/ class=summary-link><div class=article-style><p>Research is communicated more effectively and reproducibly when articles depict genetic designs consistently and fully disclose the complete sequences of all reported constructs. ACS Synthetic Biology is now providing authors with updated guidance and piloting a new tool and publication workflow that facilitate compliance with these recommended practices and standards for visual representation and data exchange.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00146 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/hillson-improving-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.6b00146 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016"><div class=card-simple><div class=article-metadata><div><span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>June 2016</span>
<span class=middot-divider></span><span class=pub-publication></span></div><a href=../publication/nguyen-converter-2016/><img src=../publication/nguyen-converter-2016/featured_hudf4a5554b30d24da91fc4bd7d3138d92_140107_600x400_fit_lanczos_2.png class=article-banner alt="A Converter from the Systems Biology Markup Language to the Synthetic Biology Open Language"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/nguyen-converter-2016/>A Converter from the Systems Biology Markup Language to the Synthetic Biology Open Language</a></h3><a href=../publication/nguyen-converter-2016/ class=summary-link><div class=article-style><p>Standards are important to synthetic biology because they enable exchange and reproducibility of genetic designs. This paper describes a procedure for converting between two standards: the Systems Biology Markup Language (SBML) and the Synthetic Biology Open Language (SBOL). SBML is a standard for behavioral models of biological systems at the molecular level. SBOL describes structural and basic qualitative behavioral aspects of a biological design. Converting SBML to SBOL enables a consistent connection between behavioral and structural information for a biological design. The conversion process described in this paper leverages Systems Biology Ontology (SBO) annotations to enable inference of a designs qualitative function.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00212 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/nguyen-converter-2016/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/acssynbio.5b00212 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jacqueline-y.-quinn/>Jacqueline Y. Quinn</a></span>, <span><a href=../author/robert-sidney-cox-iii/>Robert Sidney Cox Iii</a></span>, <span><a href=../author/aaron-adler/>Aaron Adler</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/yizhi-cai/>Yizhi Cai</a></span>, <span><a href=../author/joanna-chen/>Joanna Chen</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/nathan-j.-hillson/>Nathan J. Hillson</a></span>, <span><a href=../author/nicolas-le-novere/>Nicolas Le Novère</a></span>, <span><a href=../author/akshay-j.-maheshwari/>Akshay J. Maheshwari</a></span>, <span><a href=../author/james-alastair-mclaughlin/>James Alastair McLaughlin</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/umesh-p/>Umesh P</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/cesar-rodriguez/>Cesar Rodriguez</a></span>, <span><a href=../author/larisa-soldatova/>Larisa Soldatova</a></span>, <span><a href=../author/guy-bart-v.-stan/>Guy-Bart V. Stan</a></span>, <span><a href=../author/neil-swainston/>Neil Swainston</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>December 2015</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/quinn-sbol-2015/>SBOL Visual: A Graphical Language for Genetic Designs</a></h3><a href=../publication/quinn-sbol-2015/ class=summary-link><div class=article-style><p>Synthetic Biology Open Language (SBOL) Visual is a graphical standard for genetic engineering. It consists of symbols representing DNA subsequences, including regulatory elements and DNA assembly features. These symbols can be used to draw illustrations for communication and instruction, and as image assets for computer-aided design. SBOL Visual is a community standard, freely available for personal, academic, and commercial use (Creative Commons CC0 license). We provide prototypical symbol images that have been used in scientific publications and software tools. We encourage users to use and modify them freely, and to join the SBOL Visual community: <a href=http://www.sbolstandard.org/visual>http://www.sbolstandard.org/visual</a>.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002310" target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/quinn-sbol-2015/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1371/journal.pbio.1002310 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/meher-samineni/>Meher Samineni</a></span>, <span><a href=../author/zach-zundel/>Zach Zundel</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>December 2015</span>
<span class=middot-divider></span><span class=pub-publication><em>IEEE Life Sciences Letters</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/lib-sbo-lj-java-library-zhang-2015/>libSBOLj 2.0: A Java Library to Support SBOL 2.0</a></h3><a href=../publication/lib-sbo-lj-java-library-zhang-2015/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) is an emerging data standard for representing synthetic biology designs. The goal of SBOL is to improve the reproducibility of these designs and their electronic exchange between researchers and/or genetic design automation tools. The latest version of the standard, SBOL 2.0, enables the annotation of a large variety of biological components (e.g., DNA, RNA, proteins, complexes, small molecules, etc.) and their interactions. SBOL 2.0 also allows researchers to organize components into hierarchical modules, to specify their intended functions, and to link modules to models that describe their behavior mathematically. To support the use of SBOL 2.0, we have developed the libSBOLj 2.0 Java library, which provides an easy to use Application Programming Interface (API) for developers, including manipulation of SBOL constructs, serialization to and from an RDF/XML file format, and migration support in the form of conversion from the prior SBOL 1.1 standard to SBOL 2.0. This letter describes the libSBOLj 2.0 library and key engineering decisions involved in its design.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/lib-sbo-lj-java-library-zhang-2015/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1109/LLS.2016.2546546 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/jennifer-hallinan/>Jennifer Hallinan</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span></div><span class=article-date>December 2015</span>
<span class=middot-divider></span><span class=pub-publication><em>ACM Journal on Emerging Technologies in Computing Systems</em></span></div><a href=../publication/composable-modular-models-misirli-2015/><img src=../publication/composable-modular-models-misirli-2015/featured_hu9a585ce3f4108ffc250f57f7de15d180_818801_600x400_fit_lanczos_2.png class=article-banner alt="Composable Modular Models for Synthetic Biology"></a><h3 class="article-title mb-1 mt-3"><a href=../publication/composable-modular-models-misirli-2015/>Composable Modular Models for Synthetic Biology</a></h3><a href=../publication/composable-modular-models-misirli-2015/ class=summary-link><div class=article-style><p>Modelling and computational simulation are crucial for the large-scale engineering of biological circuits since they allow the system under design to be simulated prior to implementation in vivo. To support automated, model-driven design it is desirable that in silico models are modular, composable and use standard formats. The synthetic biology design process typically involves the composition of genetic circuits from individual parts. At the most basic level, these parts are representations of genetic features such as promoters, ribosome binding sites (RBSs), and coding sequences (CDSs). However, it is also desirable to model the biological molecules and behaviour that arise when these parts are combined in vivo. Modular models of parts can be composed and their associated systems simulated, facilitating the process of model-centred design. The availability of databases of modular models is essential to support software tools used in the model-driven design process. In this article, we present an approach to support the development of composable, modular models for synthetic biology, termed Standard Virtual Parts. We then describe a programmatically accessible and publicly available database of these models to allow their use by computational design tools.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/composable-modular-models-misirli-2015/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1145/2631921 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/bryan-bartley/>Bryan Bartley</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/michael-bissell/>Michael Bissell</a></span></div><span class=article-date>September 2015</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/bartley-bbf-2015/>BBF RFC 108: Synthetic Biology Open Language (SBOL) Version 2.0.0</a></h3><a href=../publication/bartley-bbf-2015/ class=summary-link><div class=article-style><p>The Synthetic Biology Open Language (SBOL) has been developed as a standard to support the specification and exchange of biological design information in synthetic biology, filling a need not satisfied by other pre-existing standards.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/98369 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/bartley-bbf-2015/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/zhen-zhang/>Zhen Zhang</a></span>, <span><a href=../author/tramy-nguyen/>Tramy Nguyen</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>August 2015</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/roehner-generating-2015/>Generating Systems Biology Markup Language Models from the Synthetic Biology Open Language</a></h3><a href=../publication/roehner-generating-2015/ class=summary-link><div class=article-style><p>In the context of synthetic biology, model generation is the automated process of constructing biochemical models based on genetic designs. This paper discusses the use cases for model generation in genetic design automation (GDA) software tools and introduces the foundational concepts of standards and model annotation that make this process useful. Finally, this paper presents an implementation of model generation in the GDA software tool iBioSim and provides an example of generating a Systems Biology Markup Language (SBML) model from a design of a 4-input AND sensor written in the Synthetic Biology Open Language (SBOL).</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb5003289 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/roehner-generating-2015/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb5003289 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-6 year-2015"><div class=card-simple><div class=article-metadata><div><span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/kevin-clancy/>Kevin Clancy</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/jacqueline-quinn/>Jacqueline Quinn</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>January 2015</span>
<span class=middot-divider></span><span class=pub-publication><em>Computational Methods in Synthetic Biology</em></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/myers-synthetic-2015/>The Synthetic Biology Open Language</a></h3><a href=../publication/myers-synthetic-2015/ class=summary-link><div class=article-style><p>The design and construction of engineered organisms is an emerging new discipline called synthetic biology and holds considerable promise as a new technological platform. The design of biologically engineered systems is however nontrivial, requiring contributions from a wide array of disciplines. One particular issue that confronts synthetic biologists is the ability to unambiguously describe novel designs such that they can be reengineered by a third-party. For this reason, the synthetic biology open language (SBOL) was developed as a community wide standard for formally representing biological designs. A design created by one engineering team can be transmitted electronically to another who can then use this design to reproduce the experimental results. The development and the community of the SBOL standard started in 2008 and has since grown in use with now over 80 participants, including international, academic, and industrial interests. SBOL has stimulated the development of repositories and software tools to help synthetic biologists in their design efforts. This chapter summarizes the latest developments and future of the SBOL standard and its supporting infrastructure.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1007/978-1-4939-1878-2_16 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/myers-synthetic-2015/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1007/978-1-4939-1878-2_16 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>August 2014</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/roehner-directed-2014/>Directed Acyclic Graph-Based Technology Mapping of Genetic Circuit Models</a></h3><a href=../publication/roehner-directed-2014/ class=summary-link><div class=article-style><p>As engineering foundations such as standards and abstraction begin to mature within synthetic biology, it is vital that genetic design automation (GDA) tools be developed to enable synthetic biologists to automatically select standardized DNA components from a library to meet the behavioral specification for a genetic circuit. To this end, we have developed a genetic technology mapping algorithm that builds on the directed acyclic graph (DAG) based mapping techniques originally used to select parts for digital electronic circuit designs and implemented it in our GDA tool, iBioSim. It is among the first genetic technology mapping algorithms to adapt techniques from electronic circuit design, in particular the use of a cost function to guide the search for an optimal solution, and perhaps that which makes the greatest use of standards for describing genetic function and structure to represent design specifications and component libraries. This paper demonstrates the use of our algorithm to map the specifications for three different genetic circuits against four randomly generated libraries of increasing size to evaluate its performance against both exhaustive search and greedy variants for finding optimal and near-optimal solutions.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400135t target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/roehner-directed-2014/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400135t target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014"><div class=card-simple><div class=article-metadata><div><span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/kevin-p.-clancy/>Kevin P. Clancy</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/jacqueline-y.-quinn/>Jacqueline Y. Quinn</a></span>, <span><a href=../author/cesar-a.-rodriguez/>Cesar A. Rodriguez</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/mandy-l.-wilson/>Mandy L. Wilson</a></span>, <span><a href=../author/laura-adam/>Laura Adam</a></span>, <span><a href=../author/j.-christopher-anderson/>J. Christopher Anderson</a></span>, <span><a href=../author/bryan-a.-bartley/>Bryan A. Bartley</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/joanna-chen/>Joanna Chen</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/drew-endy/>Drew Endy</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/jennifer-hallinan/>Jennifer Hallinan</a></span>, <span><a href=../author/nathan-j.-hillson/>Nathan J. Hillson</a></span>, <span><a href=../author/jeffrey-d.-johnson/>Jeffrey D. Johnson</a></span>, <span><a href=../author/allan-kuchinsky/>Allan Kuchinsky</a></span>, <span><a href=../author/matthew-lux/>Matthew Lux</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/jean-peccoud/>Jean Peccoud</a></span>, <span><a href=../author/hector-a.-plahar/>Hector A. Plahar</a></span>, <span><a href=../author/evren-sirin/>Evren Sirin</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/alan-villalobos/>Alan Villalobos</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>June 2014</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/galdzicki-synthetic-2014/>The Synthetic Biology Open Language (SBOL) provides a community standard for communicating designs in synthetic biology</a></h3><a href=../publication/galdzicki-synthetic-2014/ class=summary-link><div class=article-style><p>The synthetic biology research community describes a standard language for exchanging designs of biological &lsquo;parts&rsquo;.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/nbt.2891 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/galdzicki-synthetic-2014/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/nbt.2891 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014"><div class=card-simple><div class=article-metadata><div><span><a href=../author/evan-appleton/>Evan Appleton</a></span>, <span><a href=../author/jenhan-tao/>Jenhan Tao</a></span>, <span><a href=../author/traci-haddock/>Traci Haddock</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span></div><span class=article-date>June 2014</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/appleton-interactive-2014/>Interactive assembly algorithms for molecular cloning</a></h3><a href=../publication/appleton-interactive-2014/ class=summary-link><div class=article-style><p>Raven calculates assembly plans for complex genetic constructs from thousands of parts. It integrates user feedback on failed intermediate assemblies to improve the final outcome.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/nmeth.2939 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/appleton-interactive-2014/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/nmeth.2939 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014"><div class=card-simple><div class=article-metadata><div><span><a href=../author/d.-ewen-cameron/>D. Ewen Cameron</a></span>, <span><a href=../author/caleb-j.-bashor/>Caleb J. Bashor</a></span>, <span><a href=../author/james-j.-collins/>James J. Collins</a></span></div><span class=article-date>May 2014</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/cameron-brief-2014/>A brief history of synthetic biology</a></h3><a href=../publication/cameron-brief-2014/ class=summary-link><div class=article-style><p>In this Timeline article, Collins and colleagues chart the history of synthetic biology since its inception just over a decade ago, with a focus on both the cultural and scientific progress that has been made as well as on key breakthroughs and areas for future development.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/nrmicro3239 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/cameron-brief-2014/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/nrmicro3239 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014"><div class=card-simple><div class=article-metadata><div><span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>February 2014</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/roehner-methodology-2014/>A Methodology to Annotate Systems Biology Markup Language Models with the Synthetic Biology Open Language</a></h3><a href=../publication/roehner-methodology-2014/ class=summary-link><div class=article-style><p>Recently, we have begun to witness the potential of synthetic biology, noted here in the form of bacteria and yeast that have been genetically engineered to produce biofuels, manufacture drug precursors, and even invade tumor cells. The success of these projects, however, has often failed in translation and application to new projects, a problem exacerbated by a lack of engineering standards that combine descriptions of the structure and function of DNA. To address this need, this paper describes a methodology to connect the systems biology markup language (SBML) to the synthetic biology open language (SBOL), existing standards that describe biochemical models and DNA components, respectively. Our methodology involves first annotating SBML model elements such as species and reactions with SBOL DNA components. A graph is then constructed from the model, with vertices corresponding to elements within the model and edges corresponding to the cause-and-effect relationships between these elements. Lastly, the graph is traversed to assemble the annotating DNA components into a composite DNA component, which is used to annotate the model itself and can be referenced by other composite models and DNA components. In this way, our methodology can be used to build up a hierarchical library of models annotated with DNA components. Such a library is a useful input to any future genetic technology mapping algorithm that would automate the process of composing DNA components to satisfy a behavioral specification. Our methodology for SBML-to-SBOL annotation is implemented in the latest version of our genetic design automation (GDA) software tool, iBioSim.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400066m target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/roehner-methodology-2014/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400066m target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013"><div class=card-simple><div class=article-metadata><div><span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span></div><span class=article-date>June 2013</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/bhatia-pigeon-2013/>Pigeon: A Design Visualizer for Synthetic Biology</a></h3><a href=../publication/bhatia-pigeon-2013/ class=summary-link><div class=article-style><p>Pigeon is a Web-based tool that translates a textual description of a synthetic biology design into an image. It allows programmatic generation of design visualizations, is easy to learn, is easily extensible to new glyphs and notation, and can be connected to other software tools for visualizing their output. We present the Pigeon syntax, its current command set, and some examples of Pigeon programs and their output.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400024s target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/bhatia-pigeon-2013/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb400024s target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jason-t.-stevens/>Jason T. Stevens</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span></div><span class=article-date>May 2013</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/stevens-dynamic-2013/>Dynamic Modeling of Cellular Populations within iBioSim</a></h3><a href=../publication/stevens-dynamic-2013/ class=summary-link><div class=article-style><p>As the complexity of synthetic genetic circuits increases, modeling is becoming a necessary first step to inform subsequent experimental efforts. In recent years, the design automation community has developed a wealth of computational tools for assisting experimentalists in designing and analyzing new genetic circuits at several scales. However, existing software primarily caters to either the DNA- or single-cell level, with little support for the multicellular level. To address this need, the iBioSim software package has been enhanced to provide support for modeling, simulating, and visualizing dynamic cellular populations in a two-dimensional space. This capacity is fully integrated into the software, capitalizing on iBioSim’s strengths in modeling, simulating, and analyzing single-celled systems.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300082b target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/stevens-dynamic-2013/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300082b target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/joanna-chen/>Joanna Chen</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/timothy-s.-ham/>Timothy S. Ham</a></span>, <span><a href=../author/jay-d.-keasling/>Jay D. Keasling</a></span>, <span><a href=../author/nathan-j.-hillson/>Nathan J. Hillson</a></span></div><span class=article-date>December 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/chen-deviceeditor-2012/>DeviceEditor visual biological CAD canvas</a></h3><a href=../publication/chen-deviceeditor-2012/ class=summary-link><div class=article-style><p>Biological Computer Aided Design (bioCAD) assists the de novo design and selection of existing genetic components to achieve a desired biological activity, as part of an integrated design-build-test cycle. To meet the emerging needs of Synthetic Biology, bioCAD tools must address the increasing prevalence of combinatorial library design, design rule specification, and scar-less multi-part DNA assembly.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1186/1754-1611-6-1 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/chen-deviceeditor-2012/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1186/1754-1611-6-1 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/mandy-wilson/>Mandy Wilson</a></span>, <span><a href=../author/cesar-a.-rodriguez/>Cesar A. Rodriguez</a></span>, <span><a href=../author/matthew-r.-pocock/>Matthew R. Pocock</a></span>, <span><a href=../author/ernst-oberortner/>Ernst Oberortner</a></span>, <span><a href=../author/laura-adam/>Laura Adam</a></span>, <span><a href=../author/aaron-adler/>Aaron Adler</a></span>, <span><a href=../author/j.-christopher-anderson/>J. Christopher Anderson</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/yizhi-cai/>Yizhi Cai</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/omri-a.-drory/>Omri A. Drory</a></span>, <span><a href=../author/drew-endy/>Drew Endy</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/timothy-s.-ham/>Timothy S. Ham</a></span>, <span><a href=../author/nathan-j.-hillson/>Nathan J. Hillson</a></span>, <span><a href=../author/jeffrey-d.-johnson/>Jeffrey D. Johnson</a></span>, <span><a href=../author/allan-kuchinsky/>Allan Kuchinsky</a></span>, <span><a href=../author/matthew-w.-lux/>Matthew W. Lux</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/carlos-olguin/>Carlos Olguin</a></span>, <span><a href=../author/jean-peccoud/>Jean Peccoud</a></span>, <span><a href=../author/hector-plahar/>Hector Plahar</a></span>, <span><a href=../author/darren-platt/>Darren Platt</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/evren-sirin/>Evren Sirin</a></span>, <span><a href=../author/trevor-f.-smith/>Trevor F. Smith</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/alan-villabos/>Alan Villabos</a></span>, <span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>October 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/galdzicki-synthetic-2012/>Synthetic Biology Open Language (SBOL) Version 1.1.0</a></h3><a href=../publication/galdzicki-synthetic-2012/ class=summary-link><div class=article-style><p>In this BioBricks Foundation Request for Comments (BBF RFC), we specify the Synthetic Biology Open Language (SBOL) Version 1.1.0 to enable the electronic exchange of information describing DNA components used in synthetic biology. We define: 1. the vocabulary, a set of preferred terms and 2. the core data model, a common computational representation.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/73909 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/galdzicki-synthetic-2012/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/fusun-yaman/>Fusun Yaman</a></span>, <span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/aaron-adler/>Aaron Adler</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span></div><span class=article-date>August 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/yaman-automated-2012/>Automated Selection of Synthetic Biology Parts for Genetic Regulatory Networks</a></h3><a href=../publication/yaman-automated-2012/ class=summary-link><div class=article-style><p>Raising the level of abstraction for synthetic biology design requires solving several challenging problems, including mapping abstract designs to DNA sequences. In this paper we present the first formalism and algorithms to address this problem. The key steps of this transformation are feature matching, signal matching, and part matching. Feature matching ensures that the mapping satisfies the regulatory relationships in the abstract design. Signal matching ensures that the expression levels of functional units are compatible. Finally, part matching finds a DNA part sequence that can implement the design. Our software tool MatchMaker implements these three steps.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300032y target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/yaman-automated-2012/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300032y target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/ron-weiss/>Ron Weiss</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/aaron-adler/>Aaron Adler</a></span>, <span><a href=../author/evan-appleton/>Evan Appleton</a></span>, <span><a href=../author/jonathan-babb/>Jonathan Babb</a></span>, <span><a href=../author/swapnil-bhatia/>Swapnil Bhatia</a></span>, <span><a href=../author/noah-davidsohn/>Noah Davidsohn</a></span>, <span><a href=../author/traci-haddock/>Traci Haddock</a></span>, <span><a href=../author/joseph-loyall/>Joseph Loyall</a></span>, <span><a href=../author/richard-schantz/>Richard Schantz</a></span>, <span><a href=../author/viktor-vasilev/>Viktor Vasilev</a></span>, <span><a href=../author/fusun-yaman/>Fusun Yaman</a></span></div><span class=article-date>August 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/beal-end-end-2012/>An End-to-End Workflow for Engineering of Biological Networks from High-Level Specifications</a></h3><a href=../publication/beal-end-end-2012/ class=summary-link><div class=article-style><p>We present a workflow for the design and production of biological networks from high-level program specifications. The workflow is based on a sequence of intermediate models that incrementally translate high-level specifications into DNA samples that implement them. We identify algorithms for translating between adjacent models and implement them as a set of software tools, organized into a four-stage toolchain: Specification, Compilation, Part Assignment, and Assembly. The specification stage begins with a Boolean logic computation specified in the Proto programming language. The compilation stage uses a library of network motifs and cellular platforms, also specified in Proto, to transform the program into an optimized Abstract Genetic Regulatory Network (AGRN) that implements the programmed behavior. The part assignment stage assigns DNA parts to the AGRN, drawing the parts from a database for the target cellular platform, to create a DNA sequence implementing the AGRN. Finally, the assembly stage computes an optimized assembly plan to create the DNA sequence from available part samples, yielding a protocol for producing a sample of engineered plasmids with robotics assistance. Our workflow is the first to automate the production of biological networks from a high-level program specification. Furthermore, the workflow’s modular design allows the same program to be realized on different cellular platforms simply by swapping workflow configurations. We validated our workflow by specifying a small-molecule sensor-reporter program and verifying the resulting plasmids in both HEK 293 mammalian cells and in E. coli bacterial cells.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300030d target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/beal-end-end-2012/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1021/sb300030d target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/chris-j.-myers/>Chris J. Myers</a></span>, <span><a href=../author/tyler-patterson/>Tyler Patterson</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/jason-t.-stevens/>Jason T. Stevens</a></span>, <span><a href=../author/chris-winstead/>Chris Winstead</a></span></div><span class=article-date>June 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/madsen-design-2012/>Design and Test of Genetic Circuits Using $tt iBioSim$iBioSim</a></h3><a href=../publication/madsen-design-2012/ class=summary-link><div class=article-style><p>The simulation of biological systems prior to their physical implementation can save time, money, and potentially provide insights into alternate designs. This paper presents a simulation environment which allows for a visual design process ultimately leading to a formal model which can be efficiently simulated.</p></div></a><div class=btn-links><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/madsen-design-2012/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1109/MDT.2012.2187875 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012"><div class=card-simple><div class=article-metadata><div><span><a href=../author/karsten-temme/>Karsten Temme</a></span>, <span><a href=../author/dehua-zhao/>Dehua Zhao</a></span>, <span><a href=../author/christopher-a.-voigt/>Christopher A. Voigt</a></span></div><span class=article-date>May 2012</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/temme-refactoring-2012/>Refactoring the nitrogen fixation gene cluster from Klebsiella oxytoca</a></h3><a href=../publication/temme-refactoring-2012/ class=summary-link><div class=article-style><p>Bacterial genes associated with a single trait are often grouped in a contiguous unit of the genome known as a gene cluster. It is difficult to genetically manipulate many gene clusters because of complex, redundant, and integrated host regulation. We have developed a systematic approach to completely specify the genetics of a gene cluster by rebuilding it from the bottom up using only synthetic, well-characterized parts. This process removes all native regulation, including that which is undiscovered. First, all noncoding DNA, regulatory proteins, and nonessential genes are removed. The codons of essential genes are changed to create a DNA sequence as divergent as possible from the wild-type (WT) gene. Recoded genes are computationally scanned to eliminate internal regulation. They are organized into operons and placed under the control of synthetic parts (promoters, ribosome binding sites, and terminators) that are functionally separated by spacer parts. Finally, a controller consisting of genetic sensors and circuits regulates the conditions and dynamics of gene expression. We applied this approach to an agriculturally relevant gene cluster from Klebsiella oxytoca encoding the nitrogen fixation pathway for converting atmospheric N2 to ammonia. The native gene cluster consists of 20 genes in seven operons and is encoded in 23.5 kb of DNA. We constructed a “refactored” gene cluster that shares little DNA sequence identity with WT and for which the function of every genetic part is defined. This work demonstrates the potential for synthetic biology tools to rewrite the genetics encoding complex biological functions to facilitate access, engineering, and transferability.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.pnas.org/content/109/18/7085 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/temme-refactoring-2012/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1073/pnas.1120788109 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011"><div class=card-simple><div class=article-metadata><div><span><a href=../author/anil-wipat/>Anil Wipat</a></span>, <span><a href=../author/alan-villalobos/>Alan Villalobos</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/trevor-smith/>Trevor Smith</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span>, <span><a href=../author/nicholas-roehner/>Nicholas Roehner</a></span>, <span><a href=../author/matthew-pocock/>Matthew Pocock</a></span>, <span><a href=../author/hector-plahar/>Hector Plahar</a></span>, <span><a href=../author/jean-peccoud/>Jean Peccoud</a></span>, <span><a href=../author/chris-myers/>Chris Myers</a></span>, <span><a href=../author/goksel-misirli/>Goksel Misirli</a></span>, <span><a href=../author/curtis-madsen/>Curtis Madsen</a></span>, <span><a href=../author/matthew-lux/>Matthew Lux</a></span>, <span><a href=../author/allan-kuchinsky/>Allan Kuchinsky</a></span>, <span><a href=../author/timothy-ham/>Timothy Ham</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/john-gennari/>John Gennari</a></span>, <span><a href=../author/drew-endy/>Drew Endy</a></span>, <span><a href=../author/omri-drory/>Omri Drory</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/jacob-beal/>Jacob Beal</a></span>, <span><a href=../author/j.-christopher-anderson/>J. Christopher Anderson</a></span>, <span><a href=../author/aaron-adler/>Aaron Adler</a></span>, <span><a href=../author/laura-adam/>Laura Adam</a></span>, <span><a href=../author/cesar-rodriguez/>Cesar Rodriguez</a></span>, <span><a href=../author/mandy-wilson/>Mandy Wilson</a></span>, <span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span></div><span class=article-date>October 2011</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/wipat-title-2011/>Title: Synthetic Biology Open Language (SBOL) Version 1.0.0</a></h3><a href=../publication/wipat-title-2011/ class=summary-link><div class=article-style><p>In this BioBricks Foundation Request for Comments (BBF RFC), we specify the Synthetic Biology Open Language (SBOL) Version 1.0.0 to enable the electronic exchange of information describing DNA components used in synthetic biology. We define: 1. the vocabulary, a set of preferred terms and 2. the core data model, a common computational representation.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/66172 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/wipat-title-2011/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011"><div class=card-simple><div class=article-metadata><div><span><a href=../author/lesia-bilitchenko/>Lesia Bilitchenko</a></span>, <span><a href=../author/adam-liu/>Adam Liu</a></span>, <span><a href=../author/sherine-cheung/>Sherine Cheung</a></span>, <span><a href=../author/emma-weeding/>Emma Weeding</a></span>, <span><a href=../author/bing-xia/>Bing Xia</a></span>, <span><a href=../author/mariana-leguia/>Mariana Leguia</a></span>, <span><a href=../author/j.-christopher-anderson/>J. Christopher Anderson</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span></div><span class=article-date>April 2011</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/bilitchenko-eugene-2011/>Eugene – A Domain Specific Language for Specifying and Constraining Synthetic Biological Parts, Devices, and Systems</a></h3><a href=../publication/bilitchenko-eugene-2011/ class=summary-link><div class=article-style><p>Background Synthetic biological systems are currently created by an ad-hoc, iterative process of specification, design, and assembly. These systems would greatly benefit from a more formalized and rigorous specification of the desired system components as well as constraints on their composition. Therefore, the creation of robust and efficient design flows and tools is imperative. We present a human readable language (Eugene) that allows for the specification of synthetic biological designs based on biological parts, as well as provides a very expressive constraint system to drive the automatic creation of composite Parts (Devices) from a collection of individual Parts. Results We illustrate Eugene&rsquo;s capabilities in three different areas: Device specification, design space exploration, and assembly and simulation integration. These results highlight Eugene&rsquo;s ability to create combinatorial design spaces and prune these spaces for simulation or physical assembly. Eugene creates functional designs quickly and cost-effectively. Conclusions Eugene is intended for forward engineering of DNA-based devices, and through its data types and execution semantics, reflects the desired abstraction hierarchy in synthetic biology. Eugene provides a powerful constraint system which can be used to drive the creation of new devices at runtime. It accomplishes all of this while being part of a larger tool chain which includes support for design, simulation, and physical device assembly.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0018882" target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/bilitchenko-eugene-2011/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1371/journal.pone.0018882 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011"><div class=card-simple><div class=article-metadata><div><span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/cesar-rodriguez/>Cesar Rodriguez</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span>, <span><a href=../author/john-h.-gennari/>John H. Gennari</a></span></div><span class=article-date>February 2011</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/galdzicki-standard-2011/>Standard Biological Parts Knowledgebase</a></h3><a href=../publication/galdzicki-standard-2011/ class=summary-link><div class=article-style><p>We have created the Knowledgebase of Standard Biological Parts (SBPkb) as a publically accessible Semantic Web resource for synthetic biology (sbolstandard.org). The SBPkb allows researchers to query and retrieve standard biological parts for research and use in synthetic biology. Its initial version includes all of the information about parts stored in the Registry of Standard Biological Parts (partsregistry.org). SBPkb transforms this information so that it is computable, using our semantic framework for synthetic biology parts. This framework, known as SBOL-semantic, was built as part of the Synthetic Biology Open Language (SBOL), a project of the Synthetic Biology Data Exchange Group. SBOL-semantic represents commonly used synthetic biology entities, and its purpose is to improve the distribution and exchange of descriptions of biological parts. In this paper, we describe the data, our methods for transformation to SBPkb, and finally, we demonstrate the value of our knowledgebase with a set of sample queries. We use RDF technology and SPARQL queries to retrieve candidate “promoter” parts that are known to be both negatively and positively regulated. This method provides new web based data access to perform searches for parts that are not currently possible.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0017005" target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/galdzicki-standard-2011/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1371/journal.pone.0017005 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jean-peccoud/>Jean Peccoud</a></span>, <span><a href=../author/j.-christopher-anderson/>J. Christopher Anderson</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/douglas-densmore/>Douglas Densmore</a></span>, <span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/matthew-w.-lux/>Matthew W. Lux</a></span>, <span><a href=../author/cesar-a.-rodriguez/>Cesar A. Rodriguez</a></span>, <span><a href=../author/guy-bart-stan/>Guy-Bart Stan</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>January 2011</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/peccoud-essential-2011/>Essential information for synthetic DNA sequences</a></h3><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/nbt.1753 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/peccoud-essential-2011/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/nbt.1753 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2010"><div class=card-simple><div class=article-metadata><div><span><a href=../author/jeffrey-johnson/>Jeffrey Johnson</a></span>, <span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>December 2010</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/johnson-standard-2010/>Standard for the Electronic Distribution of SBOLv Diagrams</a></h3><a href=../publication/johnson-standard-2010/ class=summary-link><div class=article-style><p>A common method for publicly distributing design diagrams on the Web would enhance understanding of the mechanisms and intended function of synthetic DNA constructs. We aim to improve the usefulness of the depicted design to readers by establishing a systematic scheme for linking diagrams of synthetic biological mechanisms to reference information on the Registry of Standard Biological Parts [1]. Through the use of standard diagrams based on SBOL-visual (BBF RFC 16) symbols, we hope to facilitate communication about synthetic DNA constructs between research groups. This standard applies the use of the established SBOLv symbols on the Web to clearly represent synthetic constructs. Linking these diagrams using http URLs to reference information at the Registry provides a method to retrieve contextual reference information about components of DNA constructs. Use of standardized diagrams in engineering promotes fast and accurate communication between researchers, which in turn promotes effective re-use of biological parts.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/60086 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/johnson-standard-2010/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2010"><div class=card-simple><div class=article-metadata><div><span><a href=../author/j.-s.-steyn/>J. S. Steyn</a></span>, <span><a href=../author/r.-m.-boyd/>R. M. Boyd</a></span>, <span><a href=../author/y.-essa/>Y. Essa</a></span>, <span><a href=../author/p.-hall/>P. Hall</a></span>, <span><a href=../author/a.-koh/>A. Koh</a></span>, <span><a href=../author/h.-sheth/>H. Sheth</a></span>, <span><a href=../author/d.-tsu/>D. Tsu</a></span>, <span><a href=../author/s.-woodhouse/>S. Woodhouse</a></span>, <span><a href=../author/j.-hallinan/>J. Hallinan</a></span>, <span><a href=../author/a.-wipat/>A. Wipat</a></span>, <span><a href=../author/m.-r.-pocock/>M. R. Pocock</a></span></div><span class=article-date>December 2010</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/steyn-restful-2010/>A RESTful API for Supporting Automated BioBrick Model Assembly</a></h3><a href=../publication/steyn-restful-2010/ class=summary-link><div class=article-style><p>Constructing simulatable models for BioBricks by hand is a complex and time-consuming task. The time taken could be reduced by using Computer Aided Design (CAD) tools to aid in designing models, but these tools need to be augmented with domain-specific knowledge. Here we propose a standard for a RESTful (Richardson, 2007) API which facilitates the discovery and publication of models of functional biological units. This API is designed to produce parts models which can be automatically combined into complete, simulatable models of entire systems.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/60085 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/steyn-restful-2010/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2010"><div class=card-simple><div class=article-metadata><div><span><a href=../author/yuting-zheng/>Yuting Zheng</a></span>, <span><a href=../author/ganesh-sriram/>Ganesh Sriram</a></span></div><span class=article-date>May 2010</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/zheng-mathematical-2010/>Mathematical Modeling: Bridging the Gap between Concept and Realization in Synthetic Biology</a></h3><a href=../publication/zheng-mathematical-2010/ class=summary-link><div class=article-style><p>Mathematical modeling plays an important and often indispensable role in synthetic biology because it serves as a crucial link between the concept and realization of a biological circuit. We review mathematical modeling concepts and methodologies as relevant to synthetic biology, including assumptions that underlie a model, types of modeling frameworks (deterministic and stochastic), and the importance of parameter estimation and optimization in modeling. Additionally we expound mathematical techniques used to analyze a model such as sensitivity analysis and bifurcation analysis, which enable the identification of the conditions that cause a synthetic circuit to behave in a desired manner. We also discuss the role of modeling in phenotype analysis such as metabolic and transcription network analysis and point out some available modeling standards and software. Following this, we present three case studies—a metabolic oscillator, a synthetic counter, and a bottom-up gene regulatory network—which have incorporated mathematical modeling as a central component of synthetic circuit design.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.hindawi.com/journals/bmri/2010/541609/ target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/zheng-mathematical-2010/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/https://doi.org/10.1155/2010/541609 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009"><div class=card-simple><div class=article-metadata><div><span><a href=../author/cesar-rodriguez/>Cesar Rodriguez</a></span>, <span><a href=../author/suzie-bartram/>Suzie Bartram</a></span>, <span><a href=../author/anusuya-ramasubramanian/>Anusuya Ramasubramanian</a></span>, <span><a href=../author/drew-endy/>Drew Endy</a></span></div><span class=article-date>November 2009</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/rodriguez-bbf-2009/>BBF RFC 16: Synthetic Biology Open Language Visual (SBOLv) Specification</a></h3><a href=../publication/rodriguez-bbf-2009/ class=summary-link><div class=article-style><p>Synthetic Biology Open Language Visual (SBOLv) is a graphical notation that supports biological device development. It provides a formal notation for describing the physical composition of basic parts into composite parts during the development of biological devices. It is targeted for use by biological engineers in forward engineering projects. It encourages and supports model-driven engineering.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/49523 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/rodriguez-bbf-2009/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009"><div class=card-simple><div class=article-metadata><div><span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/frank-t.-bergmann/>Frank T. Bergmann</a></span>, <span><a href=../author/herbert-m.-sauro/>Herbert M. Sauro</a></span></div><span class=article-date>October 2009</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/chandran-tinkercell-2009/>TinkerCell: modular CAD tool for synthetic biology</a></h3><a href=../publication/chandran-tinkercell-2009/ class=summary-link><div class=article-style><p>Synthetic biology brings together concepts and techniques from engineering and biology. In this field, computer-aided design (CAD) is necessary in order to bridge the gap between computational modeling and biological data. Using a CAD application, it would be possible to construct models using available biological "parts" and directly generate the DNA sequence that represents the model, thus increasing the efficiency of design and construction of synthetic networks.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1186/1754-1611-3-19 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/chandran-tinkercell-2009/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1186/1754-1611-3-19 target=_blank rel=noopener>DOI</a></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009"><div class=card-simple><div class=article-metadata><div><span><a href=../author/michal-galdzicki/>Michal Galdzicki</a></span>, <span><a href=../author/deepak-chandran/>Deepak Chandran</a></span>, <span><a href=../author/alec-nielsen/>Alec Nielsen</a></span>, <span><a href=../author/jason-morrison/>Jason Morrison</a></span>, <span><a href=../author/mackenzie-cowell/>Mackenzie Cowell</a></span>, <span><a href=../author/raik-grunberg/>Raik Grünberg</a></span>, <span><a href=../author/sean-sleight/>Sean Sleight</a></span>, <span><a href=../author/herbert-sauro/>Herbert Sauro</a></span></div><span class=article-date>May 2009</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/galdzicki-provisional-2009/>Provisional BioBrick Language (PoBoL)</a></h3><a href=../publication/galdzicki-provisional-2009/ class=summary-link><div class=article-style><p>This BioBricks Foundation Request for Comments (BBF RFC) describes a semantic markup language for publishing and sharing information about BioBricks on the World Wide Web. This BBF RFC includes the recommendation for the minimal information expected when creating a Provisional BioBrick Language (PoBoL) description of BioBricks and for the implementation of the language using Web Ontology Language (OWL).</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/45537 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/galdzicki-provisional-2009/cite.bib>
Cite</button></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009"><div class=card-simple><div class=article-metadata><div><span><a href=../author/raik-grunberg/>Raik Grünberg</a></span></div><span class=article-date>April 2009</span>
<span class=middot-divider></span><span class=pub-publication></span></div><h3 class="article-title mb-1 mt-3"><a href=../publication/grunberg-bbf-2009/>BBF RFC 30: Draft of an RDF-based framework for the exchange and integration of Synthetic Biology data</a></h3><a href=../publication/grunberg-bbf-2009/ class=summary-link><div class=article-style><p>This Request for Comments (RFC) suggests a framework for the description, exchange and interlinking of Synthetic Biology data. The framework would create an open process for the evolution and “rolling” standardization of data models. It describes how data and data models are to be published, how they are exchanged and integrated between diﬀerent parties, and how they can be extended, corrected and interlinked in a decentralized fashion. These goals are achieved by embracing the RDF (Resource Description Framework), a set of W3C standards. A one-sentence summary of this proposal would there- fore be: “Use RDF according to the W3C standards.” The PoBoL pro ject (Provisional BioBrick Ontology Language, <a href=http://pobol.org>http://pobol.org</a>) is based on this idea. This RFC does not describe a data model per se but only outlines a possible architecture and rules of data exchange.</p></div></a><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://dspace.mit.edu/handle/1721.1/45143 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/grunberg-bbf-2009/cite.bib>
Cite</button></div></div></div></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by></p><p class=powered-by>Published with
<a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin=anonymous></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'event':"Events",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=../js/wowchemy.min.572b7caf7fe601716a0f74d67d754c21.js></script></body></html>